src/Stoa.ts

```diff
@@ -34,6 +34,11 @@ class Stoa {
      */
     protected task_manager: TaskManager;
 
+    /**
+     * The maximum number of blocks that can be recovered at one time
+     */
+    private _max_count_recovery: number = 64;
```

Comment:
```suggestion
    private _recovery_max_count: number = 64;
```
According to our naming convention, it is better for verbs to be defined first.

Reply:
I shortened `**max count blocks on recovery**`.